### PR TITLE
[zh-cn] sync service-accounts.md

### PR DESCRIPTION
**Pull request by hunshcn**

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
Please read through [中文本地化指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/). It's helpful to you currently.

File: content/zh-cn/docs/concepts/security/service-accounts.md [hunk 1]
```diff
@@ -474,16 +474,17 @@ API 服务器按照以下方式检查该持有者令牌的有效性：
...
 For tokens issued using the `TokenRequest` API, the API server also checks that
 the specific object reference that is using the ServiceAccount still exists,
 matching by the {{< glossary_tooltip term_id="uid" text="unique ID" >}} of that
 object. For legacy tokens that are mounted as Secrets in Pods, the API server
 checks the token against the Secret.
 -->
-TokenRequest API 为 ServiceAccount 生成**绑定令牌**。这种绑定与以该 ServiceAccount 身份运行的
-的客户端（如 Pod）的生命期相关联。
+TokenRequest API 为 ServiceAccount 生成**绑定令牌**。这种绑定与以该 ServiceAccount
+身份运行的客户端（如 Pod）的生命期相关联。有关绑定 pod 服务帐户令牌的 JWT 模式和载荷的示例，请参阅[服务账号令牌卷投射](/zh-cn/docs/tasks/configure-pod-container/configure-service-account/#serviceaccount-token-volume-projection)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/service-accounts.md:487]:
```suggestion
身份运行的客户端（如 Pod）的生命期相关联。有关绑定 Pod 服务账号令牌的 JWT 模式和载荷的示例，
请参阅[服务账号令牌卷投射](/zh-cn/docs/tasks/configure-pod-container/configure-service-account/#serviceaccount-token-volume-projection)。
```
follow context style and use a consistent term `account`

**Comment by hunshcn** [comment]
/hold
after https://github.com/kubernetes/website/pull/44557 merged

**Comment by windsonsea** [comment]
/approve

No more in-synced content with the en upstream after running `./scripts/lsync.sh` one by one.

**Review by ydFu — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**